data/headers/win32/Windows.h

```diff
@@ -0,0 +1,533 @@
+#define MAX_PATH 260
```

Comment:
We should include the Metasploit license header on top of this, so it's clear we're not actually redistributing Microsoft's headers, but instead created new ones from scratch.